### DISCUSSION TITLE
3 app crashes in 2 days

### DISCUSSION THREAD

**Discussion by SwagDad99** [OrcaSlicer/OrcaSlicer#39] [General]
Leave the app running, streaming video.  Work another model in the prepare tab.  Leave focus of the app, going to Discord, etc.  Some time later, notice that the app is no longer running.  R-launch the app, and it asks me if I want to restore the project in progress...

**3mf File for This Bug**
different projects

**To Reproduce**
Described above

**Expected behavior**
App is there when I want to return to it.

**Screenshots**
None

**Desktop (please complete the following information):**
 - Windows 11 Home
 - 32 Gig ram

**Comment by SoftFever** [comment]
Let me try to leave it opened the whole night tonight and see if I can repro it(while watching WorldCup Final ;)

**Comment by SoftFever** [comment]
I think I find the cause.
BambuSource.dll(the plugin) causes the crash.
The crash only happens when I have the camera stream on in the device tab.

```
Faulting application name: bambu-studio.exe, version: 1.0.0.0, time stamp: 0x63a13407
Faulting module name: BambuSource.dll_unloaded, version: 0.0.0.0, time stamp: 0x6396ca02
Exception code: 0xc0000005
Fault offset: 0x0000000000084ce1
Faulting process ID: 0x0x2DFC
Faulting application start time: 0x0x1D914E7D8E2E739
Faulting application path: C:\Codes\BambuStudio-SoftFever\build\BambuStudio-SoftFever_V1.4.0\bambu-studio.exe
Faulting module path: BambuSource.dll
Report ID: 8e69814f-39e1-478e-bcd6-ceb18420358f
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by julie777** [comment]
@SwagDad99 

First, you should report bugs using issues instead of discussions. They will get noticed quicker and get more responses.

I had already reported #721 for crashing. I believe I found the problem and the solution but I will not be sure until I have a few crash-free days. You can check the bug for the solution.
